src/MakePotCommand.php

```diff
@@ -293,28 +291,39 @@ protected function get_main_file_data() {
 	/**
 	 * Creates a POT file and stores it on disk.
 	 *
-	 * @return bool True on success, false otherwise.
+	 * @return Translations A Translation set.
 	 */
-	protected function makepot() {
-		$this->translations = new Translations();
+	public function extract_strings() {
```

Comment:
Note that every public method on a class is registered as a subcommand of the command. See https://make.wordpress.org/cli/handbook/commands-cookbook/#required-registration-arguments

I don't think we want this here :-) — `protected` should work fine for the sake of extensibility.-

Reply:
Ouch....
The workaround expressed in https://github.com/wp-cli/i18n-command/issues/52#issuecomment-409236464 (_attempt 4_) relies on the fact that instead of inheriting the command, I would be able to instantiate it and grab `$translations` by calling the [public] `extract_strings()` method.

Reply:
PHPDoc is used for documenting a command and perhaps making aliases. Registration is done via `WP_CLI::add_command()`.

Reply:
=> method set to protected